cdm/src/main/java/ucar/nc2/ft/SimpleGeometryStandardFactory.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 1998-2018 John Caron and University Corporation for Atmospheric Research/Unidata
```

Comment:
Can remove John Caron and replace with the person(s) who wrote this class

Reply:
Unless he helped?

Reply:
fixed